packages/apps/crm/src/hooks/use-crm-header-actions.ts

```diff
@@ -53,29 +54,81 @@ const useCatalogListActions = () => {
 const useClientDetailsActions = () => {
   const {mobileSettings} = useSelector((state: any) => state.config);
   const {client} = useSelector((state: any) => state.client);
+  const I18n = useTranslator();
 
   useEffect(() => {
     headerActionsProvider.registerModel('crm_client_details', {
       model: 'com.axelor.apps.base.db.Partner',
       modelId: client?.id,
       disableMailMessages: !mobileSettings?.isTrackerMessageEnabled,
       attachedFileScreenTitle: client?.simpleFullName,
+      actions: [
+        {
+          key: 'client-saveContact',
+          order: 10,
+          iconName: 'user-plus',
+          title: I18n.t('Crm_SaveContact'),
+          onPress: () =>
+            contactProvider.saveContact({
+              firstName: client.simpleFullName,
+              mobilePhone: client.mobilePhone,
+              fixedPhone: client.fixedPhone,
+              email: client.emailAddress?.address,
+              address: {
+                street:
+                  client.mainAddress?.addressL4 ?? client.mainAddress?.fullName,
+                country: client.mainAddress?.addressL7Country,
+                city: client.mainAddress?.primaryCity,
+                postCode: client.mainAddress?.zip,
+              },
```

Comment:
Those fields are not even fetched... It's a good idea to add them when saving contact but please when using a new field, add it to the objectFields file to fetch the informations. Otherwise it's useless to use it as information will always be null.

Same for all actions.